templatestore/frontend/src/pages/templateScreen.js

```diff
@@ -57,6 +57,10 @@ class TemplateScreen extends Component {
         this.saveTemplate = this.saveTemplate.bind(this);
         this.setMandatoryAttributes = this.setMandatoryAttributes.bind(this);
         this.onVersionAliasChange = this.onVersionAliasChange.bind(this);
+        this.getAttributes = this.getAttributes.bind(this);
+        this.buildOptions = this.buildOptions.bind(this);
```

Comment:
what is `builldOptions`

Reply:
buildOptions is a function which takes a list of values that are allowed for a particular attribute and returns options for the select element.